src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -3639,5 +3639,11 @@ In certain complex graphs or host program scenarios, Automatic mode may cause in
   <data name="PreferencesHostGenericScaleMetricUnits" xml:space="preserve">
     <value>Metric</value>
   </data>
+  <data name="CantExportWorkspaceAsImageEmptyMessage" xml:space="preserve">
+    <value>The Workspace cannot be exported as image due to it's empty.</value>
```

Comment:
@hwahlstrom Confirm if these messages would be fine from users point of view?

Reply:
@reddyashish Let's do `The workspace cannot be exported as an image because it is empty.`

Reply:
@jesusalvino Please update accordingly

Reply:
> @jesusalvino Please update accordingly

Done, both.